build_helper.sh

```diff
@@ -1,106 +0,0 @@
-#!/bin/bash
-
-SHA=$(git rev-parse HEAD | cut -c 1-7)
-BUILD=${TRAVIS_BUILD_NUMBER:-$SHA}
-
-# upstream data
-EC2_INSTANCES_INFO_COMMIT_SHA=e655e36660bc617713c4ef9a1409cc65a209cb27
-INSTANCES_URL="https://raw.githubusercontent.com/powdahound/ec2instances.info/${EC2_INSTANCES_INFO_COMMIT_SHA}/www/instances.json"
-BINDATA_FILE=generated_bindata.go
-
-LOCAL_PATH=build/s3/dv
-
-# default bucket name, can be overridden using an environment variable
-BUCKET_NAME=${BUCKET_NAME:-cloudprowess}
-
-# dependencies
-DEPS="wget git go docker"
-
-
-function clean {
-  rm -rf data
-  rm -rf ${BINDATA_FILE}
-}
-
-function check_dep {
-  command -v $1 >/dev/null 2>&1 || {
-    echo >&2 "The required dependency program '$1' is not installed. Aborting."
-    exit 1
-  }
-}
-
-function check_all_dependencies {
-  echo "Checking for the presence of the following dependencies: ${DEPS}"
-
-  for DEP in $DEPS
-  do
-    check_dep $DEP && echo "Found ${DEP}"
-  done
-  echo "All dependencies were successfully found, we're good to go!"
-}
-
-function prepare_bindata {
-  check_all_dependencies
-  go get ./...
```

Comment:
I think that's needed in case the software is cloned manually from github instead of using `go get`